plugins/modules/rabbitmq_user.py

```diff
@@ -120,6 +128,18 @@
         read_priv: .*
         write_priv: .*
     state: present
+
+# Add user to server and assign some topic permissions on / vhost.
+# The user doesn't have topic permission rules for other vhosts
+- community.rabbitmq.rabbitmq_user:
```

Comment:
```suggestion
- name: Add user to server and assign some topic permissions on / vhost, the user doesn't have topic permission rules for other vhosts
  community.rabbitmq.rabbitmq_user:
```
Or something. `- name:` statement is required in EXAMPLES according to the development doc


Also there's the sanity error:
```
00:52 >>> Standard Output
00:52 plugins/modules/rabbitmq_user.py:143:-1: traceback: SyntaxError: invalid escape sequence \.

```